tools/docker-compose/ansible/roles/sources/templates/prometheus.yml.j2

```diff
@@ -18,3 +16,16 @@ scrape_configs:
     basic_auth:
       username: admin
       password: {{ admin_password }}
+
+  - job_name: 'awx-wsrelay'
+    static_configs:
+    - targets:
+{% for i in range(control_plane_node_count|int) %}
+  {% set container_postfix = loop.index %}
+      - awx-{{ container_postfix }}:8016
+{% endfor %}
+    metrics_path: /
+    scrape_interval: {{ scrape_interval }}
+    scheme: http
+    params:
+      format: ['txt']
```

Comment:
What's this doing?

Reply:
Polling the `run_wsrelay` service for metrics. It could be a separate PR from the `awx-xxx` rename.

Exec into your awx container and run:

```
curl localhost:8016
curl localhost:8015
curl localhost:8014
```

Each service now exports metrics.

```
METRICS_SUBSYSTEM_CONFIG = {
    'server': {
        METRICS_SERVICE_CALLBACK_RECEIVER: {
            'port': 8014,
        },
        METRICS_SERVICE_DISPATCHER: {
            'port': 8015,
        },
        METRICS_SERVICE_WEBSOCKETS: {
            'port': 8016,
        },
    }
}
```

Reply:
That's fine, I just wanted a paper trail for the change.